recipes/purple-bio/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "purple-bio" %}
+{% set version = "0.4.2.1" %}
+{% set sha256 = "ad88daf6d5c7f98429b4aa4f4088245005cf26883e3ce62eb6fa900fa73b668b" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: 
+    - https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    - https://files.pythonhosted.org/packages/f3/f7/a25ef353b9fdb4d6fe2906ccd094f84c68418cbe784e1b6d22c0dc8f81b3/purple_bio-0.4.2.1.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - biopython
+    - pip
+    - python >=3
+    - tqdm
+    - pyyaml
```

Comment:
Can you try removing everything except `python` and `pip` from the host section? They're usually not needed for the actual installation.